README.md

```diff
@@ -1,92 +1,224 @@
+- [Technical Preview Features](#technical-preview-features)
+- [Python API](#python-api)
+- [Scikit-learn patching](#scikit-learn-patching)
+- [oneDAL Samples](#onedal-samples)
+- [Examples](#examples)
+- [Documentation](#documentation)
+- [oneDAL and Intel&reg; DAAL](#onedal-and-intel-daal)
 
+## Build yours high-performance data science application with intel&reg; oneDAL
 
-Intel(R) oneAPI Data Analytics Library (oneDAL) is a library that helps speed up big data analysis. 
-We provide highly optimized algorithmic building blocks for all stages of data analytics: **preprocessing**, **transformation**, **analysis**, **modeling**, **validation**, and **decision making**. Our algorithms suppost **batch**, **online**, and **distributed** processing modes of computation. 
+ intel&reg; oneDAL use all capabilities of your hardware, which allows you to get an incredible performance boost on the classic machine learning algorithms.
+We provide highly optimized algorithmic building blocks for all stages of data analytics: **preprocessing**, **transformation**, **analysis**, **modeling**, **validation**, and **decision making**.
 
 The current version of oneDAL provides Data Parallel C++ (DPC++) API extensions to the traditional C++ interface.
 
-## Table of Contents <!-- omit in toc -->
+Check out our [examples](#examples)  and [documentation](#documentation)  for information about our API
 
-- [Technical Preview Features](#preview-features)
-- [oneDAL and Intel(R) DAAL](#onedal-and-intelr-daal)
-- [Installation](#installation)
-- [Examples](#examples)
-- [Documentation](#documentation)
-- [API](#api)
+## Improve performance of your ML application
 
+oneDAL is the perfect solution if you are interested in a high-performance machine learning framework. oneDAL uses machine learning algorithms optimizations and full power of your hardware to achieve greater performance than alternative solutions offers.
 
-## Technical Preview Features 
- 
-Technical preview features are introduced to gain early feedback from developers. A preview feature is subject to change in the future releases. Using a preview feature in a production code base is therefore strongly discouraged.
-The only preview feature at the moment is `MultiNodeBatch` for K-Means, a stepless distributed algorithm based on oneCCL.
+| *Speedups of Intel&reg; oneDAL powered Scikit-learn over the original Scikit-learn, 28 cores, 1 thread/core* |
+|:--:|
+| ![](docs/readme-charts/IDP%20scikit-learn%20accelearation%20compared%20with%20stock%20scikit-learn.png) |
 
-## oneDAL and Intel(R) DAAL
 
-Intel(R) oneAPI Data Analytics Library is an extenstion of Intel(R) Data Analytics Acceleration Library (Intel(R) DAAL). 
+The size of the data is growing exponentially, as is the need for high-performance and scalable frameworks to analyze all this data and extract some benefits from it.
+Besides superior performance on a single node, the distribution mechanics of oneDAL provides excellent strong and weak scaling.
```

Comment:
Inconsistent term "distribution mechanics" with another docs

Reply:
Thanks. I replaced it with `the oneDAL distributed multi-node algorithms implementations`

Reply:
We normally use 'distributed computation mode'

Reply:
Thanks for pointing that out.  Replaced it with distributed computation mode